docs/_static/rtd_theme_overrides.css

```diff
@@ -1,3 +1,29 @@
+/* -----------------------------------------------------------------------------
+ * RTD Overrides
```

Comment:
This makes me think that RTD Override would be a good robot boss name in a mid-1990s video game trying to be cool.  Thus ends the most useful code review comment in the history of the universe.